.pre-commit-config.yaml

```
@@ -24,13 +24,13 @@ repos:
   - id: isort
 
 - repo: https://github.com/asottile/pyupgrade
-  rev: v3.7.0
+  rev: v3.8.0
   hooks:
   - id: pyupgrade
     args: ["--py36-plus"]
 
 - repo: https://github.com/asottile/setup-cfg-fmt
-  rev: v2.3.0
+  rev: v2.4.0
   hooks:
   - id: setup-cfg-fmt
```

I would usually recommend this:

```suggestion
  - id: setup-cfg-fmt
    args: [--include-version-classifiers, --min-py3-version=3.7, --max-py-version=3.11]
```

---

(or just the min version, for a package that ships binaries, it's somewhat obvious what Python versions are tested, depends if you like these or not, I do)